plugins/dell-dock/fu-plugin-dell-dock.c

```diff
@@ -128,6 +137,20 @@ fu_plugin_backend_device_added (FuPlugin *plugin,
 	if (!FU_IS_USB_DEVICE (device))
 		return TRUE;
 
+	/* GR controller internal USB HUB */
+	if ((guint) fu_usb_device_get_vid (FU_USB_DEVICE (device)) == GR_USB_VID &&
+		(guint) fu_usb_device_get_pid (FU_USB_DEVICE (device)) == GR_USB_PID) {
```

Comment:
line this up to the left bracket, otherwise it looks like the first line of the function